web/modules/custom/joinup_core/joinup_core.module

```diff
@@ -152,6 +163,23 @@ function joinup_core_form_rdf_entity_form_alter(&$form, FormStateInterface $form
 
 }
 
+/**
+ * Displays status message after the user saved the related RDF content.
+ */
+function _joinup_core_message_handler(&$form, FormStateInterface $form_state) {
```

Comment:
It's not wrong but looks so vintage, with Drupal 6 or 7 flavor, to prefix functions with underscore character :)

Also I would suggest that we use a more specific function name. For me `_joinup_core_message_handler()` is to vague/generic. What about `joinup_core_propose_message_submit()`?

Reply:
Yea this was a bit ugly I changed it. :)